### PR TITLE
Notifications page

### PR DESCRIPTION
**Pull request by Raemon**
For awhile I've wanted a view where I just read all my notifications. I have some ideas for other things to do with this view if it's better optimized (some version of it could maybe replace Recent Discussion, although there's some tricksiness of figuring out what things to display).

I plan to group them by Day.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1205634807975538) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by Raemon** [comment]
I think there are further improvements to do here but I'd rather get this version shipped for now and iterate later. (I did this as a side project over the weekend, and think it's net-positive)

File: packages/lesswrong/components/notifications/NotificationsItem.tsx [hunk 1]
```diff
@@ -58,6 +57,17 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
 });
 
+export const renderMessage = (notification: NotificationsList) => {
+  const { TagRelNotificationItem } = Components
+  switch (notification.documentType) {
+    // TODO: add case for tagRel
```
Comment (Discordius) [inline packages/lesswrong/components/notifications/NotificationsItem.tsx:63]: This TODO no longer necessary?

**Review by Discordius — APPROVED** [review]

**Comment by Raemon** [comment]
oh blarg, just realized I had gotten started on the "sort by date" and should either see that through to the next stage or revert it before merging.

**Comment by s-cheng** [comment]
@Raemon Just a warning that @agnesstenlund is working on developing a Notifications page for the EA Forum and I'm guessing we will implement it soon (like within the month). So you might prefer to wait to see what we build. One possible design as an example:

image (19)

**Pull request closed without merging**